src/fsharp/FSharp.Core/seq.fs

```diff
@@ -1087,7 +1092,7 @@ namespace Microsoft.FSharp.Collections
             checkNonNull "source" source
             seq { let hashSet = HashSet<'T>(HashIdentity.Structural<'T>)
                   for v in source do
-                      if hashSet.Add(v) then
+                      if hashSet.Add v then
```

Comment:
I'm not sure about this.  In general we add parens around dotnet interop methods.  Although not exclusively I'm not certain what convention is correct.  I'm okay with leaving this change in the PR, but you might want to think through what the convention should be.

Reply:
In the compiler we generally remove them when applied to one atomic argument (though we are inconsistent).  

There was discussion about this with @auduchinok in a recent cleanup PR - the path to consistency is to remove the parens in cases such as this - since in a very very large number of cases we would have to put them back in.

Reply:
@dsyme, I'm not really sure that I understand the comment above.

However, I believe you meant __the convention__ should be to omit the parens for a method with a single argument.

Did I parse that correctly?

Reply:
> However, I believe you meant the convention should be to omit the parens for a method with a single argument.

Yes - by "atomic" I meant no other need for parens, e.g. `x.Add v` versus `x.Add (f v)` and `x.Add (f v1, f v2)`